src/crafting.cpp

```diff
@@ -738,6 +738,8 @@ void player::start_craft( craft_command &command, const tripoint &loc )
     float best_bench_multi = 0.0;
     tripoint target = loc;
     for( const tripoint &adj : g->m.points_in_radius( pos(), 1 ) ) {
+        if( g->m.dangerous_field_at( adj ) )
+            continue;
```

Comment:
Our astyle config mandates curlies everywhere
```suggestion
        if( g->m.dangerous_field_at( adj ) ) {
            continue;
        }
```